addons/udes_stock_refactoring/tests/test_splitting.py

```diff
@@ -21,22 +22,78 @@ def setUp(self):
 
         self.picking = self.create_picking(self.picking_type_pick)
 
+        # Setup pallets and quants for apple and banana products
+        product_qty = 10
+
+        self.apple_pallet = self.create_package()
+        self.apple_quant = self.create_quant(
+            self.apple.id,
+            self.test_stock_location_01.id,
+            product_qty,
+            package_id=self.apple_pallet.id,
+        )
+
+        self.banana_pallet = self.create_package()
+        self.banana_quant = self.create_quant(
+            self.banana.id,
+            self.test_stock_location_01.id,
+            product_qty,
+            package_id=self.banana_pallet.id,
+        )
+
+    def _check_move(self, move, product, product_qty):
+        """
+        Check the supplied move line recordset has only 1 record.
+        Check the move line has the expected product and product quantity
+        """
+        self.assertEqual(len(move), 1)
+        self.assertEqual(move.product_id, product)
+        self.assertEqual(move.product_qty, product_qty)
+
+    def _check_move_line(self, move_line, product, product_qty, package):
+        """
+        Check the supplied move line recordset has only 1 record.
+        Check the move line has the expected product, product quantity,
+        package and result package
+        """
+        self.assertEqual(len(move_line), 1)
+        self.assertEqual(move_line.product_id, product)
+        self.assertEqual(move_line.product_qty, product_qty)
+        self.assertEqual(move_line.package_id, package)
+        self.assertEqual(move_line.result_package_id, package)
+
+    def _check_picking(
+        self, picking, state, group_name=False, location_id=False, location_dest_id=False
+    ):
+        """
+        Check the supplied picking recordset has only 1 record.
+        Check the picking has the expected state
+        Check group name, location and destination location if set
+        """
+        self.assertEqual(len(picking), 1)
+        self.assertEqual(picking.state, state)
+
+        if group_name:
+            self.assertEqual(picking.group_id.name, group_name)
+
+        if location_id:
+            self.assertEqual(picking.location_id, location_id)
+
+        if location_dest_id:
+            self.assertEqual(picking.location_dest_id, location_dest_id)
+
     def test_reserve_one_pallet_per_product_split(self):
         """
         Reserve self.picking with one pallet of each product and check it
         splits correctly when reserved.
         """
-        apple_pallet = self.create_package()
-        self.create_quant(
-            self.apple.id, self.test_stock_location_01.id, 10, package_id=apple_pallet.id
-        )
+        apple = self.apple
```

Comment:
Is this necessary, why not just use self.apple. We don't seem to use it that much. (No strong preference though)

Reply:
David made the same comment the other day, so have removed these now. :)